plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -537,13 +537,13 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
 
         ConnectorExpression oldExpression = constraint.getExpression();
         Map<String, String> newRegexes = new HashMap<>(handle.getRegexes());
-        List<ConnectorExpression> expressions = ConnectorExpressionUtils.extractConjuncts(constraint.getExpression());
+        List<ConnectorExpression> expressions = ConnectorExpressions.extractConjuncts(constraint.getExpression());
         List<ConnectorExpression> notHandledExpressions = new ArrayList<>();
         for (ConnectorExpression expression : expressions) {
             if (expression instanceof Call) {
                 Call call = (Call) expression;
                 // TODO Support ESCAPE character when it's pushed down by the engine
-                if ("$like_pattern".equals(call.getName()) && call.getArguments().size() == 2 &&
+                if ("$like_pattern".equals(call.getFunctionName().getFunctionName()) && call.getArguments().size() == 2 &&
```

Comment:
This doesn't check whether the function name is qualified or not.

Reply:
`new FunctionName("$like_pattern").equals(call.getFunctionName())` ?

Reply:
Not sure I fully understand the meaning of a qualified name here, is this condition better?
`call.getFunctionName().getCatalogSchemaName().isEmpty() && "$like_pattern".equals(call.getFunctionName().getFunctionName())`

Reply:
Changed + added a check to `io.trino.sql.planner.ConnectorExpressionTranslator.ConnectorToSqlExpressionTranslator#translateCall`
```           
 if (call.getFunctionName().getCatalogSchemaName().isPresent()) {
       return Optional.empty();
}
```